readme.md

```diff
@@ -680,25 +680,27 @@ A function that transform [`Response`](#response) into an array of items. This i
 ###### \_pagination.paginate
 
 Type: `Function`\
-Default: [`Link` header logic](source/index.ts)
+Default: `(response, allItems, currentItems) => getLinkFromHeaders(response)` (find details [here](source/index.ts))
 
 A function that returns an object representing Got options pointing to the next page. If there are no more pages, `false` should be returned.
 
+For example, if you want to stop when the response contains less items that expected, you should use `(response, allItems, currentItems) => currentItems.length < LIMIT ? false : { url: getNextLink(response, allItems, currentItems) }`.
```

Comment:
the `LIMIT` should be a `response.request.options.searchParams.entriesPerPage`

Reply:
`that expected` -> `than expected`

Reply:
Okay, it's updated.